lib/rules/use-colors/__tests__/index.test.js

```diff
@@ -0,0 +1,95 @@
+/*
+ Copyright 2020 Skyscanner
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+const { lint } = require('stylelint');
+
+const config = {
+  plugins: ['./index.js'],
+  rules: {
+    'backpack/use-colors': [true],
+  },
+};
+
+it('passes when setting color', async () => {
+  const {
+    results: [{ warnings }],
+  } = await lint({
+    code: 'a { color: $bpk-color-sky-blue; font-size: 14rem }',
+    config,
+  });
+
+  expect(warnings).toHaveLength(0);
+});
+
+it('passes when setting backgroundColor', async () => {
+  const {
+    results: [{ warnings }],
+  } = await lint({
+    code:
+      'a { backgroundColor: $bpk-color-sky-gray-tint-01; font-size: 14rem }',
+    config,
+  });
+
+  expect(warnings).toHaveLength(0);
+});
+
+it('should allow using an exception value', async () => {
+  const {
+    results: [{ warnings }],
+  } = await lint({
+    code: 'a { color: transparent; font-size: 14rem }',
+    config,
+  });
+
+  expect(warnings).toHaveLength(0);
+});
+
+it('gives an error and shows token to use', async () => {
+  const {
+    results: [{ warnings }],
+  } = await lint({
+    code: 'a { color: rgb(7, 112, 227); font-size: 14rem }',
+    config,
+  });
+
+  expect(warnings).toHaveLength(1);
+  expect(warnings[0].text).toBe(
+    'Use the following Backpack token instead: $bpk-color-sky-blue (backpack/use-colors)',
+  );
+});
+
+it('gives an unknown color error and to use a token', async () => {
+  const {
+    results: [{ warnings }],
+  } = await lint({
+    code: 'a { color: blue; font-size: 14rem }',
+    config,
+  });
+
+  expect(warnings).toHaveLength(1);
+  expect(warnings[0].text).toBe('Unknown color, use a Backpack token instead');
```

Comment:
nit: Can we make this more descriptive? E.g. `The color "blue" is not a valid colour.  Use a Backpack token instead`

Reply:
Not all the time would we know if the color is a valid color in our set or we know the name. I guess we could add another usecase in the code that when its a value we can identify we say it otherwise if its a `rgb(5, 200, 7)` and that does not map to a name we say unknown color.

Again this logic is the same from `eslint-plugin-backpack`

Reply:
not important to change now, it would be great to five more info but if is not simple to add then don't worry.